src/test/java/org/pesmypetcare/webservice/service/KcalAverageServiceTest.java

```diff
@@ -0,0 +1,134 @@
+package org.pesmypetcare.webservice.service;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.pesmypetcare.webservice.dao.KcalAverageDao;
+import org.pesmypetcare.webservice.entity.KcalAverageEntity;
+import org.pesmypetcare.webservice.error.DatabaseAccessException;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertSame;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.isA;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+public class KcalAverageServiceTest {
+
+    private static List<Map<String, Object>> kcalAverageAverageList;
+    private static KcalAverageEntity kcalAverageAverageEntity;
+    private static String owner;
+    private static String petName;
+    private static String date;
+    private static String date2;
+    private static Double value;
+
+    @Mock
+    private KcalAverageDao kcalAverageDao;
+
+    @InjectMocks
+    private KcalAverageService service = new KcalAverageServiceImpl();
+
+    @BeforeAll
+    public static void setUp() {
+        kcalAverageAverageList = new ArrayList<>();
+        kcalAverageAverageEntity = new KcalAverageEntity();
+        date = "2020-02-13T10:30:00";
+        date2 = "2021-02-13T10:30:00";
+        owner = "TupoJohn";
+        petName = "Perico";
+        value = 9.0 / 2.0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['9.0' is a magic number.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5400494)